android/src/main/java/com/swmansion/rnscreens/gamma/tabs/event/TabsHostNativeFocusChangeEvent.kt

```diff
@@ -9,25 +9,32 @@ class TabsHostNativeFocusChangeEvent(
     surfaceId: Int,
     viewId: Int,
     val tabKey: String,
+    val tabNumber: Int,
+    val repeatedSelectionHandledBySpecialEffect: Boolean,
 ) : Event<TabScreenDidAppearEvent>(surfaceId, viewId),
     NamingAwareEventType {
     override fun getEventName() = EVENT_NAME
 
     override fun getEventRegistrationName() = EVENT_REGISTRATION_NAME
 
-    // All events for a given view can be coalesced.
-    override fun getCoalescingKey(): Short = 0
+    override fun getCoalescingKey(): Short = (tabNumber * 10 + if (repeatedSelectionHandledBySpecialEffect) 1 else 0).toShort()
```

Comment:
What's the reasoning here? Why different key when `repeatedSelectionHandledBySpecialEffect`? 

FYI: I don't think this works anymore, as recently discovered by @t0maboro.